packages/rest-typings/src/v1/users/UsersGetNamesParamsGET.ts

```diff
@@ -0,0 +1,40 @@
+import Ajv from 'ajv';
+
+const ajv = new Ajv({
+	coerceTypes: true,
+});
+
+export type UsersGetNamesParamsGET = { userIds: string[] } | { usernames: string[] };
+
+const UsersGetNamesParamsGETSchema = {
+	anyOf: [
```

Comment:
```suggestion
	oneOf: [
```
`anyOf` allows matching with multiple schemas. In this case, there is no interesection (due to `additionalProperties` set to `false`), so we'd end up always using only one of them. Still, I think we should use `oneOf` to state clearer what is the intention here.